app/config/LoginConfig.scala

```diff
@@ -37,13 +43,25 @@ object LoginConfig {
       "replyTo" -> "core.central.production@guardian.co.uk "
     )
 
-   val switchBucket = "login-gutools-config"
-   val pandaAuthBucket = "pan-domain-auth-settings"
+    val switchBucket = "login-gutools-config"
+    val pandaAuthBucket = "pan-domain-auth-settings"
 
-   val anghammaradSnsArn = "arn:aws:sns:eu-west-1:095768028460:anghammarad-PROD-NotificationTopic-HDJHBGZT0FFD"
+    val anghammaradSnsArn = "arn:aws:sns:eu-west-1:095768028460:anghammarad-PROD-NotificationTopic-HDJHBGZT0FFD"
 
-    LoginConfig(stage, domain, host, appName, emergencyAccessTableName, tokensTableName, tokenReissueUri,
-      emailSettings, switchBucket, pandaAuthBucket, anghammaradSnsArn)
+    LoginConfig(
+      stage = stage,
+      domain = domain,
+      desktopDomain = desktopDomain,
+      host = host,
+      appName = appName,
+      emergencyAccessTableName = emergencyAccessTableName,
+      tokensTableName = tokensTableName,
+      tokenReissueUri = tokenReissueUri,
+      emailSettings = emailSettings,
+      switchBucket = switchBucket,
+      pandaAuthBucket = pandaAuthBucket,
+      anghammaradSnsArn = anghammaradSnsArn
```

Comment:
can we avoid named params for variables with the same name, to avoid noise

Reply:
it's a bit of a stylistic choice here - the values are almost all `String` so it can be easy to accidentally get the order wrong when refactoring; by using named params you can see at a glance that each argument is being passed to the right parameter, and the compiler will help you find which one is missing/superfluous if there is a mismatch

Reply:
more than persuasive